securedrop/source_templates/index.html

```diff
@@ -3,9 +3,9 @@
 
 <head>
   {% if g.organization_name == "SecureDrop" %}
-  <title>{{ g.organization_name }} | {{ gettext('Protecting Journalists and Sources') }}</title>
+  <title>Portal | {{ g.organization_name }} | {{ gettext('Protecting Journalists and Sources') }}</title>
   {% else %}
-  <title>{{ g.organization_name }} | {{ gettext('SecureDrop') }}</title>
+  <title>Portal | {{ g.organization_name }} | {{ gettext('SecureDrop') }}</title>
```

Comment:
I recommend we keep this `title` as it is, since we don't use the term "portal" anywhere else in SecureDrop's interfaces or documentation.

Reply:
Drive-by context: https://github.com/freedomofpress/securedrop/pull/6725#discussion_r1071647718 (Not dismissing what you said @cfm, only adding previous context FYI.)

Reply:
As in the previous comment above, my understanding of the guideline is that we should strive to give descriptive titles to all pages. The documentation discusses this page under the heading "Making your first submission" (https://docs.securedrop.org/en/stable/source.html), but does not specifically give the page a name. Something like "Start First Submission or Log In" would be accurate, although Technique G88 also mentions that tiles should be short (https://www.w3.org/WAI/WCAG21/Techniques/general/G88). Perhaps in could be named "New/Returning Submission"?

Reply:
A "my 2 cents"-style comment: It seems to me that "New Submission" could be a short, accurate title indeed.

I quite like "New  [...]Submission" idea and I'd argue that whether that will be the first submission you make, or you're returning to make another one... it's going to be a _new_ (if not _first_) submission in either case!

Reply:
Ah, thanks for pointing me to <https://github.com/freedomofpress/securedrop/pull/6725#discussion_r1071647718>, @gonzalo-bulnes.  I'd missed that discussion.

I appreciate the suggestion of "New Submission".  Here's the remaining case to consider:  A source returns to SecureDrop *not* to submit anything new but just to see if a journalist has replied to them.  I'm sure they'll be able to figure out how to proceed from the "First Submission" and "Return Visit" sections, but it's a flow not covered by a title like "New Submission".  Now, your original "Portal" is a neutral-enough umbrella term to include that flow, just at the cost of introducing a new term and concept we don't use elsewhere.  :-)

So I'm still inclined to keep this title as it is.  It's a special case, in `index.html` rather than via `base.html`, so it doesn't conflict with the `tab_title` block being required.  By definition, then, `index.html` will be the only page with this title, so we'll comply with the letter of WCAG 2.4.2.  And I believe we'll still comply with its spirit, since this title uniquely indicates the SecureDrop instance the source has reached.

However, if you feel strongly about giving a page-specific title here, @saffronsnail, what do you think of "Welcome"?  It's another neutral, non-jargon term I can think of (other than ["landing page"]!) that uniquely identifies what this page is *for* rather than specifying what sources can *do* on it.

["landing page"]: https://docs.securedrop.org/en/stable/deployment/landing_page.html

Reply:
I would prefer a page-specific title. I like "Welcome" because it does indicate the purpose of the page as the introduction to the site, and it's less technical than "landing page". Updated in the most recent commit.

Reply:
Yes, I meant that ["landing page"] is a reserved term we already use.  :-)  "Welcome" works well here.  Thanks!

["landing page"]: https://docs.securedrop.org/en/stable/glossary.html#landing-page